deploy/operator.yaml

```diff
@@ -1,24 +1,23 @@
 {{define "handlerPrefix"}}{{with $prefix := .HandlerPrefix}}{{$prefix | printf "%s-"}}{{end -}}{{end}}
 ---
 apiVersion: apps/v1
-kind: DaemonSet
+kind: Deployment
 metadata:
-  name: {{template "handlerPrefix" .}}nmstate-handler
+  name: {{template "handlerPrefix" .}}nmstate-webhook
   namespace: {{ .HandlerNamespace }}
+  labels:
+    app: kubernetes-nmstate
 spec:
+  replicas: 2
   selector:
     matchLabels:
-      name: {{template "handlerPrefix" .}}nmstate-handler
+      name: {{template "handlerPrefix" .}}nmstate-webhook
   template:
     metadata:
       labels:
         app: kubernetes-nmstate
-        name: {{template "handlerPrefix" .}}nmstate-handler
+        name: {{template "handlerPrefix" .}}nmstate-webhook
```

Comment:
* please rename the container from the pod as well. it is still "handler"
* I believe we don't need to set WATCH_NAMESPACE since we don't have any controller, do we? are we using POD_NAME?

Reply:
Let's remove all we can to reduce stuff and see if it works.

Reply:
WHACH_NAMESPACE is needed